rln/ffi_c_examples/main.c

```diff
@@ -16,7 +16,8 @@ int main (int argc, char const * const argv[])
 #endif
 
     if (!ffi_new_result.ok) {
```

Comment:
Naming is a bit weird here, maybe we should have something like: `_TYPE_ ffi_rln_result = ffi_rln_new();` ?